tests/mypy/mypy_function_wrapper_fn_t2.py:20: error: Argument 1 to "function_wrapper" has incompatible type "Callable[[str, Any, tuple[Any, ...], dict[str, Any]], Any]"; expected "Callable[[Callable[[VarArg(Never), KwArg(Never)], Any], Any, tuple[Any, ...], dict[str, Any]], Any]"  [arg-type]
tests/mypy/mypy_function_wrapper_fn_t2.py:27: error: Argument 1 to "__call__" of "FunctionDecorator" has incompatible type "Callable[[int, str], str]"; expected "Callable[[VarArg(Never), KwArg(Never)], Any]"  [arg-type]
Found 2 errors in 1 file (checked 1 source file)
